src/ToolBox/SOS/lldbplugin/debugclient.cpp

```diff
@@ -922,6 +921,19 @@ DebugClient::GetThreadContextById(
     dtcontext = (DT_CONTEXT*)context;
     dtcontext->ContextFlags = contextFlags;
 
+    GetContextFromFrame(frame, dtcontext);
+    hr = S_OK;
+
+exit:
+    return hr;
+}
+
+// Internal function
+void
+DebugClient::GetContextFromFrame(
+    lldb::SBFrame& frame,
```

Comment:
I get compiler errors when it is used to call the lldb API:

/ssd/coreclr/src/ToolBox/SOS/lldbplugin/debugclient.cpp:990:30: error: member function 'FindRegister' not viable: 'this' argument has type 'const lldb::SBFrame', but function is not
      marked const
    lldb::SBValue regValue = frame.FindRegister(name);


/ssd/coreclr/src/ToolBox/SOS/lldbplugin/debugclient.cpp:714:23: error: member function 'GetNumSections' not viable: 'this' argument has type 'const lldb::SBModule', but function is
      not marked const
    int numSections = module.GetNumSections();
                      ^~~~~~